types/paypal-rest-sdk/index.d.ts

```diff
@@ -0,0 +1,761 @@
+// Type definitions for paypal-rest-sdk 1.7
+// Project: https://github.com/paypal/PayPal-node-SDK
+// Definitions by: My Self <https://github.com/trainerbill>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+/// <reference types="node" />
+
+import http = require('http');
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+export as namespace paypalRestSdk;
```

Comment:
I'm pretty sure this shouldn't be here - AFAIK this is just a node module, and so does not have a UMD wrapper to run in a browser and does not make a global of itself.